### ISSUE TITLE
CollectGcBiasMetrics Interpretation

### ISSUE THREAD

**Issue by etvedte** [broadinstitute/picard#1447]
### Tool(s) involved
CollectGcBiasMetrics

### Description 
I want to document potential GC bias in Illumina libraries, but I want to confirm my interpretation of what this tool is plotting. The documentation describes normalized coverage/depth as:

"The percentage of 'coverage' or depth in a GC bin is calculated by dividing the number of reads of a particular GC content by the mean number of reads of all GC bins."

The way this is phrased is a little confusing. What is the meaning of 'reads of a particular GC content'? On reading, it appears that the tool measures GC content of the reads, which I don't think is happening. 

Rather, I interpreted the tool's functionality as:
1) slicing the genome into 100bp non-sliding windows
2) generates histogram based on count of windows in reference genome for each GC%
3) determines mean sequencing depth for all windows at each GC%
4) Divide result from 3) by mean sequencing depth for all windows across genome to get normalized depth. 

**In summary, I want to know how the numerator and the denominator of NORMALIZED_COVERAGE are calculated**

Thanks in advance!

[RDRIP_20180711_K00134_IL100105221_AGGTAGGT-Ecoli_L007_GCbiasmetrics.pdf](https://github.com/broadinstitute/picard/files/4020849/RDRIP_20180711_K00134_IL100105221_AGGTAGGT-Ecoli_L007_GCbiasmetrics.pdf)
